.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+#### Changes
+
+> #Please explain changes made in your PR#
+
+#### Screenshots
+
+> #For UI changes, please share a screenshot#
+
+#### Checklist
+
+- [ ] I have read the [contributing guidelines](http://openwisp.io/docs/developer/contributing.html#how-to-commit-your-changes-properly).
+- [ ] I have manually tested the proposed changes.
+- [ ] I have written new test cases to avoid regressions. (if necessary)
+- [ ] I have updated the documentation. (e.g. README.rst)
+
+> #Closes #(issue-number)#
```

Comment:
this shouldn't be here

Reply:
@niteshsinha17 
I you should do:
```
git fetch upstream dev
git rebase upstream/dev
git push -f
```
to fix this problem.

Reply:
I tried but it is still here, I think I should revert it

Reply:
Are you sure? It should work. 
I think this is a `Select is not broken` situation.
It should work.

I can't try fixing it because this one is open from your fork.
Can you share the output?

Reply:
Out says, already up to date.

Reply:
![image](https://user-images.githubusercontent.com/42765292/120823424-0d4e1a80-c575-11eb-8d0e-248ddd085cce.png)

Reply:
![image](https://user-images.githubusercontent.com/42765292/120823632-3f5f7c80-c575-11eb-951a-26e0c26eb501.png)